Fix casing of Arm64 etc. (#30020)
dotnet/docs: docs/core/deploying/single-file/overview.md

</PropertyGroup>
```

> ## Compress assemblies in single file app

Reviewer: ```suggestion
## Compress assemblies in single-file apps
```

> Starting with .NET 6, single file apps can be created with compression enabled on the embedded assemblies. Set `EnableCompressionInSingleFile` property to `true` to achieve this. The produced single-file will have all of the embedded assemblies compressed which can significantly reduce the size of the executable. Compression comes with a performance cost. On application start, the assemblies must be decompressed into memory, which takes some time. It's recommended to measure both the size impact and startup cost impact of enabling compression before using it as the impact varies a lot between different applications.

Reviewer: ```suggestion
Starting with .NET 6, single-file apps can be created with compression enabled on the embedded assemblies. Set the `EnableCompressionInSingleFile` property to `true` to achieve this. The single file that is produced will have all of the embedded assemblies compressed, which can significantly reduce the size of the executable. Compression comes with a performance cost. On application start, the assemblies must be decompressed into memory, which takes some time. We recommend that you measure both the size impact and startup cost impact of enabling compression before using it. The impact varies a lot between different applications.
```
